### PR TITLE
test github flow

### PR DESCRIPTION
**Pull request by atemu-yugioh**
test flow review code and merger code by leader

### CODE REVIEW

File: customer/src/services/customer-service.test.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 describe("CustomerService", () => {
+  // Which function we are testing
```
Comment (nguyen-huu-thieng) [inline customer/src/services/customer-service.test.js]: Do you think this is required here

**Review by nguyen-huu-thieng — COMMENTED** [review]
Please resolve this first.

File: customer/src/services/customer-service.test.js (hunk 1 above)
Comment (atemu-yugioh) [reply customer/src/services/customer-service.test.js]: Cool this is looks great now!

Comment (nguyen-huu-thieng) [reply customer/src/services/customer-service.test.js]: Yes this cool

**Review by nguyen-huu-thieng — APPROVED** [review]
Look great!

**Pull request merged**